src/lib.rs

```diff
@@ -52,9 +52,6 @@
 #![deny(broken_intra_doc_links)]
```

Comment:
Compiler warning:

lint `broken_intra_doc_links` has been renamed to `rustdoc::broken_intra_doc_links`

Reply:
Will address on a sepparate PR. Thanks!